docs/jwst/assign_wcs/main.rst

```diff
@@ -97,11 +98,20 @@ For each exposure, assign_wcs uses reference files and WCS header keywords
 to create the WCS object. What reference files are retrieved
 from CRDS is determined based on EXP_TYPE and other keywords in the science file header.
 
+The assign_wcs step can accept the single slope image that is the result of averaging
+over all integrations or a 3D cube of integrations in the case of TSO exposures.
 
-The assign_wcs step can accept any type of DataModel as input. In particular, for
-multiple-integration datasets the step will accept either of these data products:
-the slope results for each integration in the exposure, or the single slope image
-that is the result of averaging over all integrations.
+WCS of slitless grism exposures
+-------------------------------
+
+The WCS forward transforms for WFSS slitless grism exposures (``NIS_WFSS``, ``NRC_WFSS``, ``NRC_TSGRISM``)
```

Comment:
Remove "WFSS" here, so it's just "... for slitless grism exposures", because TSGRISM is not WFSS.

Reply:
done